code/modules/clothing/spacesuits/rig/suits/light.dm

```diff
@@ -191,6 +192,90 @@
 	)
 	species_restricted = list(SPECIES_HUMAN,SPECIES_IPC,SPECIES_RESOMI)
 
+/obj/item/rig/light/ninja/sol
+	name = "solar operative suit control module"
+	desc = "A sleek, military hardsuit with no model number or identifying logos besides a small Solar crest on its control module. Get tactical, marine."
+	suit_type = "operative"
+	icon_state = "solop_rig"
+	online_slowdown = 1
+	offline_slowdown = 2
+	initial_modules = list(
+		/obj/item/rig_module/vision,
+		/obj/item/rig_module/chem_dispenser/ninja,
+		/obj/item/rig_module/voice,
+		/obj/item/rig_module/grenade_launcher/ninja,
+		/obj/item/rig_module/ai_container,
+		/obj/item/rig_module/personal_shield,
+		/obj/item/rig_module/power_sink,
+		/obj/item/rig_module/datajack,
+		/obj/item/rig_module/self_destruct,
+		/obj/item/rig_module/cooling_unit,
+		/obj/item/rig_module/maneuvering_jets
+		)
+
+/obj/item/rig/light/ninja/gcc
+	name = "gilgamesh heavy suit control module"
+	desc = "A ridiculously bulky military hardsuit with no model number or identifying logos besides a small ICCG crest on its control module. This suit is built like a tank and weighs as much as one."
```

Comment:
lore: It peeves me that you'd bother making a stealthy black-ops suit... and still leave a tiny identifying mark anywhere. Commit to committing perfidy, dangit!